src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -22,6 +23,8 @@ public static partial class PlatformDetection
         // do it in a way that failures don't cascade.
         //
 
+        public static readonly bool IsInHelix = Environment.GetEnvironmentVariables().Keys.Cast<string>().Where(key => key.StartsWith("HELIX")).Any();
```

Comment:
This should be done lazily, similar to how other expensive properties in PlatformDetection are initialized lazily.

Reply:
I'll fix